### ISSUE TITLE
stale_after attribute not used anywhere?

### ISSUE THREAD

**Issue by louismartin** [shaypal5/cachier#95]
It seems that the stale_after attribute in BaseCore is not used anywhere.
Would it make sense to remove it as an attribute?

https://github.com/shaypal5/cachier/blob/2663f09f72a6256cfa80cfe92054949e33baa0f8/cachier/base_core.py#L16

**Comment by shaypal5** [comment]
Yeah, it's weird. If a contributor comes along, a good experiment is to remove it everywhere and then run all tests. If they pass - it's a go for a PR.

I'm guessing it's a stub for a feature I never implemented.
